interop-cats/shared/src/main/scala/zio/interop/Schedule.scala

```diff
@@ -421,7 +421,7 @@ object Schedule {
    * @see zio.ZSchedule.succeedLazy
    */
   final def succeedLazy[F[+_], A](a: => A): Schedule[F, Any, A] =
-    new Schedule(ZSchedule.succeedLazy(a))
+    new Schedule(ZSchedule.succeed(a))
```

Comment:
I guess this function should be 
1. implemented as `forever.as(a)` to preserve previous laziness
2. deprecated

Reply:
Oh right

Reply:
Same here, `succeed` is already lazy

Reply:
No, it will evaluate on call - https://github.com/zio/zio/blob/master/core/shared/src/main/scala/zio/ZSchedule.scala#L727 - although it's not important now if even zio-core's `succeedLazy` is now also strict anyway